src/main/java/mekanism/common/world/DummyWorld.java

```diff
@@ -1,6 +1,7 @@
 package mekanism.common.world;
 
 import java.io.File;
+import javax.annotation.Nonnull;
```

Comment:
Could probably just use `ParametersAreNonnullByDefault` here as that's what thrown on the package by Forge